plugins/modules/cloud/misc/terraform.py

```diff
@@ -95,6 +95,11 @@
     description:
       - A group of key-values to provide at init stage to the -backend-config parameter.
     required: false
+  backend_config_file:
+    description:
+      - The path to a configuration file to provide at init state to the -backend-config parameter.
+        This can accept a list of pathes to multiple configuration files.
+    required: false
```

Comment:
`required: false` is not necessary (its the default). You should always specify `type` instead.
```suggestion
    type: list
    elements: path
```